<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt快速开发框架生成器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="projectSettingsGroupBox">
      <property name="title">
       <string>项目设置</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="projectPathLabel">
         <property name="text">
          <string>项目路径：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="projectPathLineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browseButton">
           <property name="text">
            <string>浏览...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="projectNameLabel">
         <property name="text">
          <string>项目名称：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="projectNameLineEdit">
         <property name="text">
          <string>MyProject</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="projectTypeLabel">
         <property name="text">
          <string>项目类型：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="projectTypeComboBox">
         <item>
          <property name="text">
           <string>QWidget</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QMainWindow</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="useFormCheckBox">
         <property name="text">
          <string>使用Form文件</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="filePreviewGroupBox">
      <property name="title">
       <string>文件名预览</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="headerFileLabel">
         <property name="text">
          <string>头文件：</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="headerFilePreviewLabel">
         <property name="text">
          <string>myproject.h</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sourceFileLabel">
         <property name="text">
          <string>源文件：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="sourceFilePreviewLabel">
         <property name="text">
          <string>myproject.cpp</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="formFileLabel">
         <property name="text">
          <string>Form文件：</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="formFilePreviewLabel">
         <property name="text">
          <string>myproject.ui</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="additionalOptionsGroupBox">
      <property name="title">
       <string>附加选项</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="addResourcesCheckBox">
         <property name="text">
          <string>添加资源文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="qt6CompatibilityCheckBox">
         <property name="text">
          <string>兼容Qt6</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="processGroupBox">
         <property name="title">
          <string>进程模式</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="multiInstanceRadioButton">
            <property name="text">
             <string>允许进程有多个实例（使用Application类）</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="singleInstanceRadioButton">
            <property name="text">
             <string>允许进程只有一个实例（使用SingleApplication类）</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useStylesheetCheckBox">
         <property name="text">
          <string>使用样式表</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="useIconFontCheckBox">
         <property name="text">
          <string>使用Icon字体库</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="previewButton">
        <property name="text">
         <string>框架预览</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="createButton">
        <property name="text">
         <string>创建</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
